frappe/public/js/frappe/socketio_client.js

```diff
@@ -43,7 +43,7 @@ frappe.socket = {
 		});
 
 		$(document).on("form_refresh", function(e, frm) {
-			if (frm.is_new()) {
+			if (cur_frm.is_new()) {
```

Comment:
Let's fix this by passing `frm` instead of using cur_frm. @MaxMorais do you want to do this?